eng/common/pipelines/templates/steps/sparse-checkout.yml

```diff
@@ -8,9 +8,13 @@ parameters:
       - Name: $(Build.Repository.Name)
         Commitish: $(Build.SourceVersion)
         WorkingDirectory: $(System.DefaultWorkingDirectory)
+  - name: MultipleCheckout
```

Comment:
I'd suggest calling this something like SkipDefaultCheckout.

Reply:
K.

Reply:
@weshaggard  done.